### PR TITLE
modify testcontainer

### PR DESCRIPTION
**Pull request by AlexStocks**
### Purpose or design rationale of this PR

*Describe your change. Make sure to answer these three questions: What does this PR do? Why does it do it? How does it do it?*

modify testcontainer

### PR title

Your PR title must follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) (as we are doing squash merge for each PR), so it must start with one of the following [types](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type):

- [ ] build: Changes that affect the build system or external dependencies (example scopes: yarn, eslint, typescript)
- [ ] ci: Changes to our CI configuration files and scripts (example scopes: vercel, github, cypress)
- [ ] docs: Documentation-only changes
- [ ] feat: A new feature
- [ ] fix: A bug fix
- [ ] perf: A code change that improves performance
- [x] refactor: A code change that doesn't fix a bug, or add a feature, or improves performance
- [ ] style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
- [ ] test: Adding missing tests or correcting existing tests

### Deployment tag versioning

Has `tag` in `common/version.go` been updated or have you added `bump-version` label to this PR?

- [x] No, this PR doesn't involve a new deployment, git tag, docker image tag
- [ ] Yes

### Breaking change label

Does this PR have the `breaking-change` label?

- [x] No, this PR is not a breaking change
- [ ] Yes

### CODE REVIEW

File: common/testcontainers/testcontainers.go [hunk 1]
```diff
@@ -106,6 +110,62 @@ func (t *TestcontainerApps) StartL2GethContainer() error {
 	return nil
 }
 
+// StartPoSL1Container starts the PoS L1 container by running the associated Docker Compose configuration
+func (t *TestcontainerApps) StartPoSL1Container() error {
+	var (
```
Comment (georgehao) [inline common/testcontainers/testcontainers.go:115]: How about remove the `StartL1GethContainer`? @colinlyguo

Comment (colinlyguo) [reply common/testcontainers/testcontainers.go:115]: make sense. let's remove `StartL1GethContainer`, using post-pos L1 instead.

Comment (georgehao) [reply common/testcontainers/testcontainers.go:115]: after the discussion, this will changed in next pr

**Review by georgehao — DISMISSED** [review]

File: common/testcontainers/docker-compose.yml [hunk 2]
```diff
@@ -8,8 +8,8 @@ services:
               mkdir -p /data/execution &&
               cp -a /execution/* /data/execution/"
     volumes:
-      - ${HOST_PATH:-../../..}/common/docker-compose/l1/consensus:/consensus
-      - ${HOST_PATH:-../../..}/common/docker-compose/l1/execution:/execution
+      - ${HOST_PATH:-../..}/common/testcontainers/consensus:/consensus
+      - ${HOST_PATH:-../..}/common/testcontainers/execution:/execution
```
Comment (colinlyguo) [inline common/testcontainers/docker-compose.yml]: HOST_PATH can also be removed.

**Review by colinlyguo — DISMISSED** [review]

**Review by colinlyguo — APPROVED** [review]

**Review by georgehao — APPROVED** [review]

**Pull request merged**